curriculum/challenges/english/01-responsive-web-design/basic-css/understand-absolute-versus-relative-units.md

```diff
@@ -37,7 +37,7 @@ assert(
 Your `red-box` class should give elements 1.5em of `padding`.
 
 ```js
-assert(code.match(/\.red-box\s*?{[\s\S]*padding:\s*?1\.5em/gi));
+assert(code.match(/\.red-box\s*?{[\s\S]*padding\s*:\s*?1\.5em/gi));
```

Comment:
Hey @SethFalco, this one also seems to miss a semi-colon.

Reply:
Matching an optional semi-colon or not matching a semi-colon at all would result in the same behavior.
The last item in a CSS rule doesn't have to have a semi-colon, which is where the user is encouraged to add it, so this is valid.

```css
.red-box {
  background-color: red;
  margin: 20px 40px 20px 40px;
  padding: 1.5em
}
```

I can change it if you want, but _in my opinion_ for as long as we're using regex, I think this is better. Once we move to a CSS parser/helper, then maybe we can be more pedantic about it?